modules/javafx.base/src/main/java/com/sun/javafx/property/adapter/Disposer.java

```diff
@@ -43,16 +43,16 @@
  * of the associated Runnable object will be called.
  */
 public class Disposer implements Runnable {
-    private static final ReferenceQueue queue = new ReferenceQueue();
-    private static final Map<Object, Runnable> records = new ConcurrentHashMap<>();
+    private static final ReferenceQueue<Object> QUEUE = new ReferenceQueue<>();
+    private static final Map<Reference<?>, Runnable> RECORDS = new ConcurrentHashMap<>();
```

Comment:
I don't really care for the suggestion that most IDEs seem to offer of naming `static final` objects (as opposed to primitive types) using all caps. This should be left up to the developer, and in this case I think the existing names are fine and don't need to be changed. I note that this code was borrowed from Java2D where a similar fix was done to add the generics, but the names were left unchanged.

Reply:
I only use all-uppercase if the collection itself is immutable. Here, the collections/wrapper are mutable, so I wouldn't change the names either. A matter of style.

Reply:
I reverted these changes.